docs/release.md

```diff
@@ -0,0 +1,81 @@
+Release process
+===============
+
+The goal of this document is to define the release process of KubeVirt, but it
+is not intended to define any release criteria.
+
+
+Overview
+--------
+- KubeVirt uses [semantic versioning](http://semver.org)
+- Primary artefact is the source tree in form of a signed tap using
+  [git evtag](https://github.com/cgwalters/git-evtag)
+- Binary artifacts are built using automation
+- The releases appear on a time based release schedule
+
+
+Content
+-------
+The primary artifact of a release is the source tree itself. The trust on the
+tree is established by using _git-evtag_ which can be used to sign the tree
+recursively, including blobs and submodules.
+
+For convenience a number of binary artifacts can be provided alongside a
+release, those are:
+
+- Container images (currently docker images)
```

Comment:
Shouldn't the images be published on a registry and tagged? Adding one big manifest file (for the beginning non-ha, not so secure, ...) might be a better choice. Here is an example of a weave release with yamls included. Maybe that makes sense for us.

Reply:
This sentence should not imply that we publish the containers as binaries/tarballs. I rather intend to say that the release is the source code, but we provide aartifacts - like containers and bianries - in the relevant channels. Thus containers on dockerhub and binaries on the github release page.
I added a sentence to make this clearer.

WRT manifests - For now they are part of the repo.
And yes, we want to also publish them in a dedicated and compiled form for releases - I'm also working on something on that side.